tileserv/tools/time.py

```diff
@@ -0,0 +1,22 @@
+from zoneinfo import ZoneInfo
+from datetime import date, datetime, time
+
+
+def pin_issue_date_to_time(issue_date: date, issue_time: time):
+    """ Pin the time of an issue date to given time"""
+    return datetime.combine(issue_date, issue_time, tzinfo=ZoneInfo('America/Vancouver'))
+
+
+def morning_time(issue_date: date):
+    """ Pin the time of an issue date to morning time, 08:00 """
+    return pin_issue_date_to_time(issue_date, time(8, 0, 0))
+
+
+def noon_time(issue_date: date):
+    """ Pin the time of an issue date to noon time, 13:00 """
+    return pin_issue_date_to_time(issue_date, time(13, 0, 0))
+
+
+def evening_time(issue_date: date):
+    """ Pin the time of an issue date to evening time, 16:45 """
```

Comment:
```suggestion
    """ Pin the time of an issue date to evening time, 17:45 """
```

Reply:
atm I don't see where/how these functions are going to be used. tif_pusher.py seems to only use an issue _date_, not a time. I'm presuming that comes up in a future PR?

Reply:
Yeah tif pusher will decorate the input date with times for uploading to API and ultimately end up as `run_datetime`